compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/java/KSAnnotationJavaImpl.kt

```diff
@@ -97,6 +97,16 @@ class KSAnnotationJavaImpl private constructor(val psi: PsiAnnotation) : KSAnnot
         presentValueArguments.plus(argumentsFromDefault)
     }
 
+    override val defaultArguments: List<KSValueArgument> by lazy {
```

Comment:
This shares quite a lot same logic as `fun AnnotationDescriptor.getDefaultArguments`, can you make this share the logic? A way to do this is to make `getDefaultArguments()` an extension function of KotlinType.